prql-js/Cargo.toml

```diff
@@ -1,12 +1,18 @@
 [package]
 description = "Javascript bindings for prql-compiler"
-edition = "2021"
-license = "Apache-2.0"
 name = "prql-js"
 publish = false
-repository = "https://github.com/PRQL/prql"
+
+edition.workspace = true
+rust-version.workspace = true
 version.workspace = true
 
+# Unsupported by `wasm-pack` (which seems kinda unmaintained now...)
+# license.workspace = true
+# repository.workspace = true
+license = "Apache-2.0"
+repository = "https://github.com/PRQL/prql"
```

Comment:
I'm curious, what is it wasm-pack is doing?  Looking at the dependencies, it doesn't look like it should be directly parsing these files; it uses `cargo-metadata` which relies on the `cargo-metadata` command, so the relevant version of cargo is used to parse the files.

Reply:
Hmmm, I'm not sure. This is the error though:

```
# from prql-js

❯ wasm-pack build --target bundler
Error: failed to parse manifest: .../prql/prql-js/Cargo.toml
Caused by: invalid type: map, expected a string for key `package.license`
```

Reply:
Ah, I missed `toml` in their dependency list.  Yup, they are manually processing things.